lib/dao.js

```diff
@@ -2407,6 +2577,190 @@ DataAccessObject.prototype.unsetAttribute = function unsetAttribute(name, nullif
 };
 
 /**
+ * Replace set of attributes.
+ * Performs validation before replacing.
+ *
+ * @trigger `validation`, `save` and `update` hooks
+ * @param {Object} data Data to replace
+ * @param {Object} [options] Options for replace
+ * @param {Function} cb Callback function called with (err, instance)
+ */
+DataAccessObject.prototype.replaceAttributes = function(data, options, cb) {
+  var Model = this.constructor;
+  var id = getIdValue(this.constructor, this);
+  var inst = this;
+  var strict = this.__strict;
+  return Model.replaceById(id, inst, Model, data, options, strict, cb);
+};
+
+DataAccessObject.replaceById = function(id, inst, Model, data, options, strict, cb) {
```

Comment:
@Amir-61 using my example code above, you can access `strict` via `var strict = inst.__strict;`